tests/unit/test-util/test.adb.js

```diff
@@ -1310,4 +1311,32 @@ describe('utils/adb', () => {
     });
   });
 
+  describe('test exposed adb API functions', () => {
+    it('test listADBDevices', async () => {
+      const stubDiscoverDevices = sinon.stub(
+        ADBUtils.prototype, 'discoverDevices'
+      );
+      stubDiscoverDevices.resolves(['emulator1', 'device2']);
+      const promise = listADBDevices();
+      const devices = await assert.isFulfilled(promise);
+      assert.deepEqual(devices, ['emulator1', 'device2']);
+    });
+
+    it('test listADBFirefoxAPKs', async () => {
+      const adb = getFakeADBKit({});
+      const stubDiscoverInstalledFirefoxAPKs = sinon.stub(
+        ADBUtils.prototype, 'discoverInstalledFirefoxAPKs'
+      );
+      stubDiscoverInstalledFirefoxAPKs
+        .resolves(['package1', 'package2', 'package3']);
+      const promise = listADBFirefoxAPKs('device1');
+      const packages = await assert.isFulfilled(promise);
+      sinon.assert.calledWith(
+        new ADBUtils({adb}).discoverInstalledFirefoxAPKs,
+        'device1'
+      );
```

Comment:
I think this can (and should) be actually asserting on `stubDiscoverInstalledFirefoxAPKs` and so this one should be:
```
sinon.assert.calledWith(
        stubDiscoverInstalledFirefoxAPKs,
        'device1'
      );
```

and we should also be able to remove `const adb = getFakeADBKit({});` that is at line 1326, because it is going to not be needed after this change.

Let me know if for any reason this change doesn't work and the test fails (but I think that it should work just fine).